funnel/templates/past_projects_section.html.jinja2

```diff
@@ -0,0 +1,20 @@
+{% for project in past_projects %}
+  <tr>
+    <td data-th="Date">
```

Comment:
Is `gettext` being called in JS? Else this should have a `{% trans %}` right here.